canvas_modules/common-canvas/src/icons/icon.scss

```diff
@@ -41,7 +41,7 @@
 	}
 }
 
-svg.properties-icon, svg.canvas-icon {
+svg.properties-icon, svg.canvas-icon, .toolbar-item > svg {
```

Comment:
Aren't we forcing consumer to have this color if we add this?  Shouldn't this come from the harness instead?  Do we know why the color is wrong if the first place?

Reply:
The color is now being replaced because the "run flow" button is actually ghost button instead of just a wrapped <SVG> like some of the other toolbar items. The updated carbon component has built in color styling that is being passed on to the SVG. I can remove this and let the consumer decide

Reply:
moved this to the test harness `.scss`. I put it under App.scss in the harness since it seemed to be the closest fit out of the css files